paper-slider.html

```diff
@@ -543,6 +543,10 @@
         this._setRatio(this._calcRatio(this.immediateValue));
 
         this.$.sliderKnob.style.left = (this.ratio * 100) + '%';
+        if (this.dragging) {
+          this._knobstartx = this.ratio * this._w || 0;
```

Comment:
The `|| 0` is copied from [line 574](https://github.com/jscissr/paper-slider/blob/patch-1/paper-slider.html#L574).